spec/features/participant-declarations/submit_participant_declarations_spec.rb

```diff
@@ -0,0 +1,178 @@
+    and_the_lead_provider_submits_a_declaration_for_the_participant_using_the_same_unique_id(@npq_profile_id, "npq-leading-teaching")
+    then_the_declaration_made_is_valid
+    and_the_lead_provider_receives_a_response_to_confirm_that_the_declaration_was_successful
+  end
+
+  scenario "NPQ participant details sent to provider, declaration sent using same unique ID, errors exist in declaration" do
+    given_an_npq_participant_has_been_entered_onto_the_dfe_service
+    when_the_participant_details_are_passed_to_the_lead_provider
+    and_the_lead_provider_submits_a_declaration_for_the_participant_using_the_same_unique_id_with_error("npq-leading-teaching")
+    then_the_declaration_made_is_invalid
+    and_the_lead_provider_receives_a_response_to_confirm_that_the_declaration_has_a_validation_error
+  end
+
+  scenario "NPQ participant details sent to provider, declaration sent using different unique ID, errors exist in declaration" do
+    given_an_npq_participant_has_been_entered_onto_the_dfe_service
+    when_the_participant_details_are_passed_to_the_lead_provider
+    and_the_lead_provider_submits_a_declaration_for_the_participant_using_the_same_unique_id("111-222-333-444", "npq-leading-teaching")
+    then_the_declaration_made_is_invalid
+    and_the_lead_provider_receives_a_response_to_confirm_that_the_declaration_has_a_validation_error
+  end
+
+private
+
+  def given_an_early_career_teacher_has_been_entered_onto_the_dfe_service
+    ect_profile = create(:early_career_teacher_profile)
+    delivery_partner = create(:delivery_partner)
+    create(:partnership,
+           school: ect_profile.school,
+           lead_provider: @cpd_lead_provider.lead_provider,
+           cohort: ect_profile.cohort,
+           delivery_partner: delivery_partner)
+
+    @participant_id = ect_profile.user.id
+  end
+
+  def given_an_ecf_mentor_has_been_entered_onto_the_dfe_service
+    @mentor_id = create(:mentor_profile).user.id
+  end
+
+  def given_an_npq_participant_has_been_entered_onto_the_dfe_service
+    npq_lead_provider = create(:npq_lead_provider, cpd_lead_provider: @cpd_lead_provider)
+    npq_course = create(:npq_course, identifier: "npq-leading-teaching")
+    @npq_profile_id = create(:npq_validation_data,
+                             npq_lead_provider: npq_lead_provider,
+                             npq_course: npq_course).user.id
+  end
+
+  def when_the_participant_details_are_passed_to_the_lead_provider
+    @session.get("/api/v1/participants",
+                 headers: { "Authorization": "Bearer #{@token}" })
+    participants = JSON.parse(@session.response.body).dig("data").map { |participant| participant["id"] }
+    expect(participants.first).to eq(@participant_id)
+  end
+
+  def and_the_lead_provider_submits_a_declaration_for_the_participant_using_the_same_unique_id(participant_id, course_identifier)
+    params = common_params(participant_id, course_identifier)
+    submit_request(params)
+  end
```

Comment:
Should we split it into three methods to avoid the parameters?
Say
```
and_the_lead_provider_submits_a_declaration_for_the_ect_using_their_id
and_the_lead_provider_submits_a_declaration_for_the_mentor_using_their_id
and_the_lead_provider_submits_a_declaration_for_the_npq_using_their_id
```